### PR TITLE
Note in config.sample.php that certain previews are not available in ms ...

### PR DESCRIPTION
**Pull request by DeepDiver1975**
...windows

### CODE REVIEW

**Comment by DeepDiver1975** [comment]
looks good :+1:

**Comment by LukasReschke** [comment]
I think this will make the first block disappear from the documentation.

@MorrisJobke Right?

File: config/config.sample.php [hunk 1]
```diff
@@ -658,6 +658,18 @@
  *  - OC\Preview\SVG
  *  - OC\Preview\TIFF
  */
+ 
+/**
```
Comment (MorrisJobke) [inline config/config.sample.php]: Can the two comment paragraphs be combined. Otherwise the above one would create a new section ;)

**Comment by MorrisJobke** [comment]
I will fix that

**Comment by MorrisJobke** [comment]
:+1:

**Comment by MorrisJobke** [comment]
@carlaschroder @DeepDiver1975 Is this also valid for stable7?

**Comment by LukasReschke** [comment]
@MorrisJobke Yes

**Comment by MorrisJobke** [comment]
stable7 5f70ed1

**Pull request merged**